osbs/build/build_request.py

```diff
@@ -867,7 +889,13 @@ def render(self, validate=True):
                            # add the path to the plugin's
                            # configuration. This is done elsewhere.
                            None):
-                          self.spec.registry_secrets.value})
+                          self.spec.registry_secrets.value,
+
+                          ('buildstep_plugins', 'orchestrate_build', 'osbs_client_config'):
```

Comment:
@lcarva, IMHO it's fine to merge this PR before the plugin exists; as @rcerven said, if the plugin is not in the build request (because it doesn't exist yet) it won't be configured.

The only dependency is exactly what the parameter name is, and that's something we can just agree on before merge.